File: Practical/HTTP_Example/NodeJS_Clients/GET_QUERY.js
const axios = require('axios')

// Program to make a simple GET request and parse the Response Body
const HOST = "http://localhost:";
const PORT = "3000"
const PATH = "/sensor/query"

const REQUEST_Query = {
    params:{
        id: 1
    }
}
// Make a GET request to Server
axios.get(HOST + PORT + PATH, REQUEST_Query)
    .then((response)=>{
        const RESPONSE_BODY  = JSON.stringify(response.data,null,2);
        console.log("Data: "+RESPONSE_BODY+". Response Code: "+ response.status)
    }).catch(error=>{
        console.log(error);
    })

File: Practical/HTTP_Example/Python_Clients/GET_QUERY.py
# Program to perform simple GET request to HTTP server
import requests
import json

HOST = "http://localhost:"
PORT = "3000"
PATH = "/sensor/query"

REQUEST_QUERY = {
    "id": 1
}

response = requests.get(HOST + PORT + PATH, params=REQUEST_QUERY)

print(response.json())